enterprise/backend/test/metabase_enterprise/serialization/api_test.clj

```diff
@@ -25,20 +25,25 @@
       (GzipCompressorInputStream.)
       (TarArchiveInputStream.)))
 
+(def FILE-TYPES
+  [#"([^/]+)?/$"                               :dir
```

Comment:
Not a fan of `flatland.ordered.map`?

Reply:
I'm builtins guy! 😁

Clojure should be more eager adding useful stuff in the core >_>